src/validators/mod.rs

```diff
@@ -82,7 +82,8 @@ impl PySome {
     }
 
     #[classmethod]
-    pub fn __class_getitem__(cls: &PyType, _args: &PyAny) -> Py<PyType> {
+    pub fn __class_getitem__(cls: &PyType, item: &PyAny) -> Py<PyType> {
+        let _ = item;
         cls.into_py(cls.py())
```

Comment:
Hmm what's going on here? I'm sure there's a good reason, I'm just confused by it.

Reply:
Good challenge - `item` is unused, I think probably it's better if I put an `#[allow(unused_variables)]` in front.

Reply:
Can’t you just name the parameter `_item`?

Reply:
For Rust's side, yes, but PyO3 doesn't try to be smart about whether you want the leading `_` or not so then the Python argument would also be `_item`.

I want to keep the name `item` on the Python side because that's what PEP 560 used in https://peps.python.org/pep-0560/#class-getitem